src/main/kotlin/io/github/nomisrev/routes/profile.kt

```diff
@@ -0,0 +1,49 @@
+package io.github.nomisrev.routes
+
+import arrow.core.Either
+import arrow.core.raise.either
+import io.github.nomisrev.IncorrectJson
+import io.github.nomisrev.service.ProfileService
+import io.ktor.http.HttpStatusCode
+import io.ktor.resources.Resource
+import io.ktor.server.application.ApplicationCall
+import io.ktor.server.application.call
+import io.ktor.server.resources.get
+import io.ktor.server.routing.Route
+import io.ktor.server.util.getOrFail
+import io.ktor.util.pipeline.PipelineContext
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.MissingFieldException
+import kotlinx.serialization.Serializable
+
+@JvmInline @Serializable value class GetProfile(val username: String)
```

Comment:
This `GetProfile` is redundant, it's being used in line 33 but also immediately unwrapped in the same line.

Reply:
Addressed in [f1a373f](https://github.com/nomisRev/ktor-arrow-example/pull/186/commits/f1a373fbe767a64c6d72fd20a7d362f2e059091d)